doc/api/worker_threads.md

```diff
@@ -252,6 +252,111 @@ if (isMainThread) {
 }
 ```
 
+## `worker.postMessageToWorker(destination, value[, transferList][, timeout])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+* `destination` {number} The target thread ID.
+* `value` {any} The value to send.
+* `transferList` {Object\[]} If one or more `MessagePort`-like objects are passed in `value`,
+  a `transferList` is required for those items or [`ERR_MISSING_MESSAGE_PORT_IN_TRANSFER_LIST`][] is thrown.
+  See [`port.postMessage()`][] for more information.
+* `timeout` {number} Time to wait for the message to be delivered in milliseconds.
+  By default it's `undefined`, which means wait forever.
```

Comment:
This should indicate what happens if the message times out.

Reply:
Done, I also added the `ERR_WORKER_MESSAGING_TIMEOUT` error.